exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/Metadata.java

```diff
@@ -927,15 +927,11 @@ public void setMax(Object max) {
     @JsonProperty List<ParquetFileMetadata_v2> files;
     @JsonProperty List<String> directories;
     @JsonProperty String drillVersion;
-    @JsonProperty boolean isDateCorrect;
+    @JsonProperty int writerVersion;
```

Comment:
Not sure I completely follow... Here's what I understood:

1. The class is used when ready a Parquet file footer. (In such a case, the writer version would presumably be the actual version from the footer, or null if the file is old, was not written by Drill, etc.)
2. The class is used when reading metadata, but then some fields are not used.
3. In the case where the metadata is for one file (only) is the writer version stored in the metadata? So, object created when reading metadata will have the same information as the original file?
4. If the metadata is for multiple files, then the writer version is never set? (Or, should the rule be that the writer version is set if all files have the same version?)

Since this is a bit hard to understand, it would be very helpful to describe all this in a Javadoc comment for the class.